software-engineering-policies/CloudDevelopment/CloudDevelopmentPolicy.md

```diff
@@ -103,6 +103,38 @@ A Trivy Stage is included in the previously mentioned Terraform deployment templ
 
 ### Secrets Management
 
+#### Centralized Secrets Management
+
+All secrets, credentials, and sensitive data used by applications must be stored in Azure Key Vault.
```

Comment:
Is it possible to define 'sensitive data' further by enumerating things that are considered sensitive?